calculator.rb

```diff
@@ -0,0 +1,64 @@
+# prints welcome message
+puts "Welcome to the Calculator! Which operator would you like to use?"
+puts "1. add(+)"
+puts "2. subtract(-)"
+puts "3. multiply(*)"
+puts "4. divide(/)"
+puts "5. compute exponent(^)"
+puts "6. compute modulo(%)"
+print "Please enter an operator (name or symbol): "
+
+# asks the user to enter an operator, prints error if invalid input
+oper = gets.chomp.downcase
+until ["+","add","-","subtract","*", "multiply","/","divide","^","compute exponent", "%", "compute modulo"].include?(oper) == true
+  print "You entered an invalid operator. Please enter a new operator: "
+  oper = gets.chomp
+end
+
+# asks the user to enter two numbers, prints error if invalid input
```

Comment:
You've written almost exactly the same code here twice, to get the first number and the second number. Could you DRY that up by putting this logic in a method?